src/cta/index.html

```diff
@@ -2,20 +2,22 @@
 <html>
 <head></head>
 <body id="main">
-        <div class="ra21-button" role="button" data-href="" id="idpbutton">
-            <div class="identityprovider ra21-button-logo-wrap">
-                <img class="ra21-button-logo" src="<%= require('../assets/sa-white.svg') %>">
-            </div>
-            <div class="identityprovider d-flex justify-content-center align-items-center text-truncate ra21-button-text">
-                <div class="text-truncate">
-                    <div class="ra21-button-text-primary text-truncate" id="headline"></div>
-                    <div class="ra21-button-text-primary text-truncate" id="title">Access through your institution</div>
+        <button class="ra21-button" type="button" id="idpbutton">
```

Comment:
<div> was replaced with a <button> to make it "tab-able" in the browser, and type="button" was inserted to prevent default <button> behavior. ID was left in the code to connect it back to the Javascript.